File: GoLang/7 kyu/Highest and lowest/main.go
package main

import (
	"strings"
	"strconv"
	"fmt"
)

func main() {
	fmt.Println(HighAndLow("1 2 -3 4 5"))
}
  
func HighAndLow(in string) string {
	result := ""
	inSplit := strings.Fields(in)
	var res []int
	
	for i:= 0; i < len(inSplit); i++ {
	  intVar, _ := strconv.Atoi(string(inSplit[i]))
	  res = append(res, intVar)
	}
	
	max := res[0]
	min := res[0]
	for i := 0; i < len(res); i++ {
	  if max < res[i] { max = res[i] }
	  if res[i] < min { min = res[i] }
	}
	
	result = strconv.Itoa(max) + " " + strconv.Itoa(min)
	return result
}

File: Python/7 kyu/Highest and lowest/main.py
def high_and_low(numbers):
    nums = ""
    arr = numbers.split()
    max = arr[0]
    for i in arr: 
        if int(i) > int(max):
            max = i 
    min = arr[0]
    for k in arr: 
        if int(k) < int(min):
            min = k 
    nums = max + " " + min
    return nums

if __name__ == "__main__":
    print(high_and_low("1 2 -3 4 5"))